web/src/routing/routes/ResetPassword/ResetPassword.test.tsx

```diff
@@ -0,0 +1,118 @@
+import { Router } from "react-router-dom";
+import { ThemeProvider } from "@features/ThemeProvider/ThemeProvider";
+import ResetPassword from "./ResetPassword";
+import { render, screen } from "@testing-library/react";
+import { createMemoryHistory } from "history";
+import { Auth } from "aws-amplify";
+import { act } from "react-dom/test-utils";
+import userEvent from "@testing-library/user-event";
+
+const renderWithHistory = (
+  initialEntries: string[] = [],
+  Component: JSX.Element
+) => {
+  const history = createMemoryHistory({
+    initialEntries,
+  });
+
+  return {
+    ...render(
+      <ThemeProvider>
+        <Router history={history}>{Component}</Router>
+      </ThemeProvider>
+    ),
+    history,
+  };
+};
+
+const expectDocumentElements = () => {
```

Comment:
`expectDocumentElements` sounds like assertions. IMO it shouldn't be used to retrieve DOM elements. I see 2 solutions:

1. `expectDocumentElements` does not return anything and is used only like an assertion (rather this is not what you want)
2. `expectDocumentElements` is renamed to `getResetPasswordElements`, and everything stays as it is (besides `expect(...)` which are not needed with `getBy...`, as `getBy...` has got assertion included already